### PR TITLE
Update with latest team revamp

### PR DESCRIPTION
**Pull request by aturon**

### CODE REVIEW

File: _data/team.yml [hunk 1]
```diff
@@ -147,24 +165,34 @@ teams:
...
+    responsibility: "overall direction for tools for working with Rust code"
+    members: [japaric, brson, nrc, michaelwoerister, killercup, jonathandturner]
+    lead: nrc
+  - name: Dev tools peers
+    responsibility: "oversight of specific Rust tools, and coordination with dev tools team"
+    members: [alexcrichton, emilio, fitzgen, manishearth, llogiq, oli-obk, matklad, jwilm, imperio, misdreavous, steveklabnik]
+    lead: nrc
+  - name: Cargo team
+    responsibility: "design and implementation of Cargo"
+    members: [alexcrichton, carols10cents, withoutboats, matklad, wycats, aturon]
```
Comment (QuietMisdreavus) [inline _data/team.yml]: Does the Cargo team have a team lead?

File: _data/team.yml [hunk 2]
```diff
@@ -48,14 +49,19 @@ people:
   dotdash:
     name: Björn Steinbrink
     irc: doener
+    ex-teams: ["compiler"]
```
Comment (steveklabnik) [inline _data/team.yml]: wait this is a thing? i have some PRs to send...

File: _data/team.yml (hunk 1 above)
Comment (steveklabnik) [inline _data/team.yml]: line below this says @aturon

Comment (QuietMisdreavus) [inline _data/team.yml]: Must have been added in after I made the comment. When it was first posted it didn't have that line.

File: _data/team.yml [hunk 3]
```diff
@@ -147,24 +165,35 @@ teams:
...
-    responsibility: "Rust tools (e.g. Cargo, rustup)"
-    members: [japaric, brson, nrc, alexcrichton, vadimcn, wycats, carols10cents, michaelwoerister]
-    lead: alexcrichton
+  - name: Dev tools team
+    responsibility: "overall direction for tools for working with Rust code"
+    members: [japaric, brson, nrc, michaelwoerister, killercup, jonathandturner]
+    lead: nrc
+  - name: Dev tools peers
+    responsibility: "oversight of specific Rust tools, and coordination with dev tools team"
+    members: [alexcrichton, emilio, fitzgen, manishearth, llogiq, oli-obk, matklad, jwilm, imperio, misdreavous, steveklabnik]
```
Comment (QuietMisdreavus) [inline _data/team.yml]: Oh, just saw when it was rendered - @GuillaumeGomez and I are in this list via our github usernames, not our irc handles.

Comment (QuietMisdreavus) [inline _data/team.yml]: Also, the team listing on the Community page wasn't updated with the new teams.

**Pull request merged**